### ISSUE TITLE
`undefined` is output before results if there is no config file present

### ISSUE THREAD

**Issue by tom-fletcher** [JamieMason/syncpack#106]
## Description

If there is no config file present, when running `syncpack` (with any command) `undefined` is output to the console before the results are displayed. When a config file is present, results display as normal.

Example output:

```
$ pnpm syncpack                   
undefined
- @esbuild-kit/esm-loader ^2.5.4
- ...
```

## Suggested Solution

The offending line is in `disk.ts`:

https://github.com/JamieMason/syncpack/blob/b3ffc1c92a70a3b56214b74d84b5cce5063aa097/src/lib/disk.ts#L43

I suspect this may be a bit of debug code that has managed to creep in and needs deleted?

**Comment by JamieMason** [comment]
Thanks a lot @tom-fletcher, I'd been meaning to sort this but not had chance – thanks a lot for finding the cause for me. I think you're right that I'd left some debug code in. 👍🏻

**Comment by JamieMason** [comment]
This is finally released in [8.5.14](https://github.com/JamieMason/syncpack/releases/tag/8.5.14), thanks a lot.

**Comment by tom-fletcher** [comment]
Nice - as ever, thanks for the package and the work that goes into maintaining it!
